### DISCUSSION TITLE
JSON array -  data how to create data ?

### DISCUSSION THREAD

**Discussion by NuttasitNew** [prisma/prisma#23847] [Q&A]
### Question

i have payload JSON like this
```
{
  "orderId":1712936861121,
  "customerID":6,
  "phone":"0000000000",
  "serviceId":[1,10,8,11,9],
  "discount":0,
  "travelCost":0,
  "employeeId":[1,3,2,4]
}
```
how i use prisma to create this payload to postgreSQL 
my project use nextjs 14

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @NuttasitNew 👋 

Thank you for raising this question.

Based on the payload you shared, I'd like to ask some clarifying questions.

1. Does this payload represent the shape of an `Order` ?

2. Assuming the payload represents an `Order` model, Is there a relationship between `Order` and `Services` represented by `serviceId` or `Order` and `Employee` represented by `employeeId`?

3. If the payload doesn't belong to an `Order` model, What model does it belong to or do you want to store the payload using a [json field](https://www.prisma.io/docs/orm/prisma-client/special-fields-and-types/working-with-json-fields)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
